pkg/asset/agent/manifests/agentclusterinstall.go

```diff
@@ -104,5 +105,17 @@ func (a *AgentClusterInstall) Load(f asset.FileFetcher) (bool, error) {
 	}
 	a.Config = agentClusterInstall
 
+	if err = a.finish(); err != nil {
+		return false, err
+	}
 	return true, nil
 }
+
+func (a *AgentClusterInstall) finish() error {
+
+	if a.Config == nil {
+		return errors.New("missing configuration or manifest file")
```

Comment:
With this PR I'm seeing this error when my cluster-manifests directory is missing:

`
FATAL failed to fetch Agent Installer ISO: failed to fetch dependency of "Agent Installer ISO": failed to fetch dependency of "Agent Installer Ignition": failed to fetch dependency of "Agent Manifests": failed to generate asset "Agent PullSecret": missing configuration or manifest file 
`

Would it be possible to include the path of the missing file(s) in the error message?

Reply:
I avoided explicitly to specify the exact file path mainly because as soon as we'll enable the `Generate()` code - for the all the agent manifests assets - this check will not be required anymore, and the `Load()` will become optional (either the `Load()` will find some file or the `Generate()` will be able to create a new resource, in any case the shared `finish()` method will receive non-nil data to validate/manipulate. But if you think it will be more clear, even in this transitory period, I could specify it as well.